RIOT-OS/RIOT
# cpu/cc26x0: rename to cc26x0_cc13x0

### Contribution description

Renames the cc13x0 CPU (as I did before with cc26x2 cpu) in order to add support to the cc13x0 family of micro controllers.

### Testing procedure

cc26x0 boards should still compile and work:

- `make -C examples/hello-world BOARD=cc2650-launchpad`
- `make -C examples/hello-world BOARD=cc2650stk`


### Issues/PRs references
